### ISSUE TITLE
[BUG]: Cannot find module '../engine/IReport'

### ISSUE THREAD

**Issue by petrvorobev** [IBMa/equal-access#1527]
### Project

a11y checker

### Browser

Chrome

### Operating system

Windows

### Description

Trying to run an exmaple throws an error. 

Node.js v18.16.1

C:\DEV\Research\equal-access\accessibility-checker\boilerplates\mocha-zombie>node test/basic.test.js
node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module '../engine/IReport'
Require stack:
- C:\DEV\Research\equal-access\accessibility-checker\boilerplates\mocha-zombie\node_modules\accessibility-checker\lib\common\report\ReporterManager.js
- C:\DEV\Research\equal-access\accessibility-checker\boilerplates\mocha-zombie\node_modules\accessibility-checker\lib\common\config\ACConfigManager.js
- C:\DEV\Research\equal-access\accessibility-checker\boilerplates\mocha-zombie\node_modules\accessibility-checker\lib\ACBrowserManager.js
- C:\DEV\Research\equal-access\accessibility-checker\boilerplates\mocha-zombie\node_modules\accessibility-checker\index.js
- C:\DEV\Research\equal-access\accessibility-checker\boilerplates\mocha-zombie\test\basic.test.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (C:\DEV\Research\equal-access\accessibility-checker\boilerplates\mocha-zombie\node_modules\accessibility-checker\lib\common\report\ReporterManager.js:67:17)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (C:\DEV\Research\equal-access\accessibility-checker\boilerplates\mocha-zombie\node_modules\accessibility-checker\lib\common\config\ACConfigManager.js:62:25)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\DEV\\Research\\equal-access\\accessibility-checker\\boilerplates\\mocha-zombie\\node_modules\\accessibility-checker\\lib\\common\\report\\ReporterManager.js',
    'C:\\DEV\\Research\\equal-access\\accessibility-checker\\boilerplates\\mocha-zombie\\node_modules\\accessibility-checker\\lib\\common\\config\\ACConfigManager.js',
    'C:\\DEV\\Research\\equal-access\\accessibility-checker\\boilerplates\\mocha-zombie\\node_modules\\accessibility-checker\\lib\\ACBrowserManager.js',
    'C:\\DEV\\Research\\equal-access\\accessibility-checker\\boilerplates\\mocha-zombie\\node_modules\\accessibility-checker\\index.js',
    'C:\\DEV\\Research\\equal-access\\accessibility-checker\\boilerplates\\mocha-zombie\\test\\basic.test.js'
  ]
}

### Steps to reproduce

1. Clone repository.
2. Go to equal-access\accessibility-checker\boilerplates\mocha-zombie
3. run npm install
4. run node test/basic.test.js

**Comment by tombrunet** [comment]
@petrvorobev There were some issues with .gitignore files with the npm deployment last week preventing some files from deploying (including IReport). Those particular errors should be resolved through the hotfixes we deployed (3.1.57).

That said, not sure that particular boilerplate will run anyway. There are some other issues with zombie because it doesn't support a full DOM API. It isn't clear to me how many people still use it. I'll leave this issue open to take another look at that particular boilerplate.

**Comment by ErickRenteria** [comment]
Merged to master
